src/main/java/com/google/cloud/genomics/utils/grpc/VariantUtils.java

```diff
@@ -168,6 +190,36 @@ public int compare(Variant v1, Variant v2) {
           BY_ALTERNATE_BASES)));
 
   /**
+   * A Comparator that orders {@code Variant} objects by chromosome, position, and alleles.
+   */
+  public static final Comparator<Variant> CHROMOSOMAL_ORDER = new ChromosomalOrderComparator();
+
+  private static final class ChromosomalOrderComparator implements Comparator<Variant> {
+    @Override
+    public int compare(Variant v1, Variant v2) {
+      int comparison =
+          ComparisonChain.start()
```

Comment:
Where is `ComparisonChain` declared?  Your Travis builds are failing because of that.